Backend/Tournament_User/TournamentUserMVSC/server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const app = express();
+const tournamentUserRoutes = require('./routes/tournamentUserRoutes');
+const errorHandler = require('./middlewares/errorHandler');  // Import error-handling middleware
+const successHandler = require('./middlewares/successHandler');// Import success-handling middleware
+
+
+// Middleware to parse incoming JSON
+app.use(express.json());
+
```

Comment:
You might need to add CORS for frontend to be able to call. Try to hide ORIGIN in an env var